toolset/benchmark/framework_test.py

```diff
@@ -74,7 +74,7 @@ def is_accepting_requests(self):
 
         url = "http://%s:%s%s" % (self.benchmarker.config.server_host,
                                   self.port,
-                                  self.runTests[test_type].get_url())
+                                  self.runTests["plaintext"].get_url())
```

Comment:
This change breaks all tests.

Reply:
Ok then how can I change which test to use for this check for the ReactPHP bench mark?